### PR TITLE
Add ORDER BY to some tests to have consistent output

### PR DESCRIPTION
**Pull request by byucesoy**

### CODE REVIEW

File: src/test/regress/expected/multi_metadata_sync.out [hunk 1]
```diff
@@ -228,7 +228,7 @@ SELECT * FROM pg_dist_shard ORDER BY shardid;
  mx_testing_schema.mx_test_table | 1310007 | t            | 1610612736    | 2147483647
 (8 rows)
 
-SELECT * FROM pg_dist_shard_placement ORDER BY shardid;
+SELECT * FROM pg_dist_shard_placement ORDER BY shardid, nodename, nodeport;
```
Comment (aamederen) [inline src/test/regress/expected/multi_metadata_sync.out]: It wouldn't make much difference here, since there is only one placement per shard here.

Comment (metdos) [inline src/test/regress/expected/multi_metadata_sync.out]: Let's keep the good practice :)

**Comment by metdos** [comment]
Looks good, :shipit: after travis passes.

File: src/test/regress/expected/multi_metadata_sync.out (hunk 1 above)
Comment (aamederen) [inline src/test/regress/expected/multi_metadata_sync.out]: I just don't see any point on changing the deterministic parts. It disagree with touching queries returning 0 rows, 1 row or multiple rows with guaranteed order.

If you insist on this, at least we can change it to `ORDER BY placementid`.

**Pull request merged**